superset/sql_lab.py

```diff
@@ -171,6 +171,8 @@ def handle_error(msg):
     # Limit enforced only for retrieving the data, not for the CTA queries.
     superset_query = SupersetQuery(rendered_query)
     executed_sql = superset_query.stripped()
+    SQL_MAX_ROWS = app.config.get('SQL_MAX_ROW', None)
```

Comment:
Also `app.config.get('SQL_MAX_ROW', None)` is equivalent to `app.config.get('SQL_MAX_ROW')` as the second slot is only used to return a non-None value if the key doesn't exist.

Reply:
Done